### ISSUE TITLE
Incorrect accessibilityPath positions in ScrollView (iOS)

### ISSUE THREAD

**Issue by nwdmtech** [software-mansion-labs/react-native-enriched-markdown#696]
**Describe the bug**
In our project we use `<EnrichedMarkdownText />` inside of a `<ScrollView  />` which renders a chat bot conversation. 
When new elements are being rendered in that conversation and you start VoiceOver on iOS the focus ring for the text of `<EnrichedMarkdownText />` are not rendered at the right position and thus not properly being scrolled to when navigating through the elements.

**To Reproduce**
Render `<EnrichedMarkdownText />` inside of a `<ScrollView  />` and dynamically add new `<EnrichedMarkdownText />` elements to the scroll view container.

**Expected behavior**
When the screen reader reads out the content of a text element inside of `<EnrichedMarkdownText />` the focus ring should be located at the right position and it should be scrolled to the right position.

**Screenshots**
<img width="1073" height="425" alt="Image" src="https://github.com/user-attachments/assets/1100cf52-b11c-47bf-98cc-8c1b2e911853" />

**Additional context**
The patch in the attached screenshot actually fixed the issue for us. So I would assume the issue here is that the paths are being cached and not recalculated?

**Device (please complete the following information):**
 - Device: iPhone17
 - OS: iOS26
 - Version: 1.0.1

**Comment by eszlamczyk** [comment]
Hi @nwdmtech thanks for the bug report! I'll try looking into it next week

**Comment by nwdmtech** [comment]
@eszlamczyk thanks for the fix!
Do you have a timeline for the next release?

**Comment by eszlamczyk** [comment]
@nwdmtech we are expecting release next week
